_posts/2022-03-16-aws-wafv2-association-with-api-gateway.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title: "AWS WAFV2 association with API Gateway"
+date: 2022-03-16
+tags: AWS API WAFV2
+author: Dale Murugan
+image: img/lock.png
+---
+
+I recently had to attach a Web Application Firewall (WAF) regional Access Control List (ACL) to an API gateway created using the [Serverless Framework](https://www.serverless.com/). The only quality documentation I could find was from our very own [Natalie Laing](https://au.linkedin.com/in/natalie-laing-652a9131) in this [post](https://mechanicalrock.github.io/2019/09/26/aws-waf-association-with-api-gateway.html) she wrote back in 2019. I ran into an issue where my WebACL would not properly associate to the API. After some research I found that many others also faced this issue, mainly because the ‘Classic WAF’ has been deemed [outdated](https://docs.aws.amazon.com/waf/latest/developerguide/classic-waf-chapter.html) by AWS.
```

Comment:
```suggestion
I recently had to attach a Web Application Firewall (WAF) regional Access Control List (ACL) to an API gateway created using the [Serverless Framework](https://www.serverless.com/). The only quality documentation I could find was from our very own [Natalie Laing](https://au.linkedin.com/in/natalie-laing-652a9131) in this [post](https://mechanicalrock.github.io/2019/09/26/aws-waf-association-with-api-gateway.html) she wrote back in 2019. I ran into an issue where my WebACL would not properly associate to the API. After some research I found that many others also faced this issue, mainly because the ‘Classic WAF’ has been [deprecated](https://docs.aws.amazon.com/waf/latest/developerguide/classic-waf-chapter.html) by AWS.
```

Better wording?